src/tests/ftest/util/agent_utils.py

```diff
@@ -65,8 +153,22 @@ def run_agent(basepath, server_list, client_list=None):
     sessions = {}
     user = getpass.getuser()
 
-    # if empty client list, 'self' is effectively client
-    client_list = include_local_host(client_list)
+    # if empty client list, 'test' is effectively client
+    if client_list is None:
+        client_list = include_local_host(client_list)
+    client_count = len(client_list)
+
+    # Create the DAOS Agent configuration yaml file to pass
+    # with daos_agent -o <FILE_NAME>
+    agent_yaml = os.path.join(test.tmp, "daos_agent.yml")
+    agent_config = DaosAgentConfig()
+    agent_config.get_params(test)
+    agent_config.hostlist.value = client_list
+
+    access_point = server_list[0] + ":" + str(agent_config.port)
```

Comment:
For combining strings it is best to use the `join()` method:
```suggestion
    access_point = ":".join(server_list[0], str(agent_config.port))
```